third_party/bazel/cuda_static/cuda_static_configure.bzl

```diff
@@ -2,16 +2,20 @@ load("//bazel:common.bzl", "files_exist", "get_bash_bin")
 
 _TF_CUDA_HOME = "TF_CUDA_HOME"
 
+def _create_dummy_repository(repo_ctx):
+    repo_ctx.symlink(Label("//bazel/cuda_static:BUILD"), "BUILD")
```

Comment:
The `BUILD` file under `bazel/cuda_static` indicate that the directory is a sub-module under current workspace. It HAS NOTHING TO DO with the generated workspace. This happens to work just because they are both empty. Pls use another file or make `cuda_static.BUILD.tpl` work for this case.

Reply:
done